test/cases/api.sh

```diff
@@ -438,6 +438,36 @@ EOF
 )
 export FIREWALL_CUSTOMIZATION_BLOCK
 
+if [ "$TEST_MODULE_HOTFIXES" = 1 ]; then
+  if [ "$ARCH" = "x86_64"]; then
+    NGINX_REPO_URL="https://rpmrepo.osbuild.org/v2/mirror/public/el8/el8-x86_64-nginx-20231207"
+  else
+    NGINX_REPO_URL="https://rpmrepo.osbuild.org/v2/mirror/public/el8/el8-aarch64-nginx-20231207"
+  fi
```

Comment:
nitpick:  You could also just use
```bash
NGINX_REPO_URL="https://rpmrepo.osbuild.org/v2/mirror/public/el8/el8-${ARCH}-nginx-20231207"
```

Reply:
I did, but the repo url validator complains xD